akka-remote/src/main/scala/akka/remote/artery/tcp/ArteryTcpTransport.scala

```diff
@@ -152,19 +152,13 @@ private[remote] class ArteryTcpTransport(_system: ExtendedActorSystem, _provider
         }
       }
 
-      if (streamId == ControlStreamId) {
-        // restart of inner connection part important in control flow, since system messages
-        // are buffered and resent from the outer SystemMessageDelivery stage.
-        RestartFlow.withBackoff[ByteString, ByteString](
-          settings.Advanced.OutboundRestartBackoff,
-          settings.Advanced.GiveUpSystemMessageAfter, 0.1)(flowFactory)
```

Comment:
this max of `GiveUpSystemMessageAfter` must have been a thinko, no reason to exponentially increase the backoff that much